metrics/metric_template.go

```diff
@@ -46,12 +46,15 @@ func (mv *Vars) Compose(providedTemplate string) (string, error) {
 }
 
 func (mv *Vars) Parse(stream *Stream) {
+	space, _ := stream.App.Space()
```

Comment:
The tests run by Travis are failing. It looks like we need to mock API responses for `Space()` and `Org()`.